stdlib/public/runtime/Metadata.cpp

```diff
@@ -6158,16 +6157,10 @@ void swift::blockOnMetadataDependency(MetadataDependency root,
 #if !SWIFT_STDLIB_PASSTHROUGH_METADATA_ALLOCATOR
 
 namespace {
-  struct alignas(2 * sizeof(uintptr_t)) PoolRange
-      : swift::aligned_alloc<2 * sizeof(uintptr_t)> {
```

Comment:
I think this needs rebasing off the alignment PR?

Reply:
> I think this needs rebasing off the alignment PR?

Was waiting to merge that one, hence this one being a WIP. Updated.